protos/flyteidl/core/workflow.proto

```diff
@@ -78,6 +78,9 @@ message NodeMetadata {
     oneof interruptible_value {
         bool interruptible = 6;
     };
+
+    // Total wait time a workflow can be delayed by queueing.
+    string max_queue_time = 7;
```

Comment:
Can we use [duration](https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/duration.proto) instead?